e2e_tests/tests/streaming/test_client.py

```diff
@@ -0,0 +1,79 @@
+import random
+
+import pytest
+
+from determined.common import streams
+from determined.common.api import bindings
+from determined.common.streams import _client
```

Comment:
You are importing the modules, yes, but in the case of the `streams._client` module, there is nothing that calling code should be importing directly.

The full code layout rfc is [here](https://github.com/rb-determined-ai/pedl/blob/9bd2168ec23fbba866194f3d7d3cfc243d267100/python_code_layout.md) if you want to read it.

TL;DR is: you can generally assume that any of our python files that start with an underscore are not meant to be imported directly, and the relevant symbols should be available in their parent namespace.

So in this case, all the symbols you get from `_client.*` should actually be accessed as `stream.*`.

You can think of this as a way to separate the organization of source code (which code goes in which `_`-prefixed files) from the import path for accessing exported symbols.